app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Let's change the association between a user and a like to suit the project requirement. The ERD shows `author_id` instead of `user_id`. We want a like to belong to an `author`.
HINT: `belongs_to :author, class_name: 'User'`

Reply:
Thanks, done!